service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataService.java

```diff
@@ -56,7 +57,7 @@ public RoboticsData prepareRoboticsData(final CaseData caseData, final Long case
         return RoboticsData.builder()
             .caseNumber(caseData.getFamilyManCaseNumber())
             .applicationType(deriveApplicationType(caseData.getOrders()))
-            .feePaid(2055.00)
+            .feePaid(isNotEmpty(caseData.getAmountToPay()) ? toDouble(caseData.getAmountToPay()) : 2055.00)
```

Comment:
remember that the amountToPay is in pence, is the robot expecting it this way?

Reply:
I believe so. @Gogetter, is this right?

Reply:
@KiranVarma02 we would want to pound value as sending in pence will be wrong.

Reply:
thanks @Gogetter for help. Done.